teams/team-geordi/dev/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Did you just copy over the chart from the github repo? We don't need to do it, we can just point the team-geordi application to the github repository. That is if we don't need to customize anything.

Reply:
Yes i was thinking on same lines but i thought having a teams and having the code in a team like `team-geordi` was mandatory. Im aligned now. Now im pointing directly from repo.